prow/config.yaml

```diff
@@ -592,28 +592,28 @@ presubmits:
   - name: pull-security-kubernetes-kubemark-e2e-gce
     always_run: true
     context: pull-security-kubernetes-kubemark-e2e-gce
-    rerun_command: "@k8s-bot pull-security-kubernetes-kubemark-e2e-gce test this"
-    trigger: "@k8s-bot (pull-security-kubernetes-kubemark-e2e-gce |kubemark e2e )?test this"
+    rerun_command: "/test pull-security-kubernetes-kubemark-e2e-gce"
+    trigger: "(^@k8s-bot (pull-security-kubernetes-kubemark-e2e-gce |kubemark e2e )?test this\r?$|^/test( pull-security-kubernetes-kubemark-e2e-gce| kubemark e2e)?\r?$)"
 
   - name: pull-security-kubernetes-kubemark-e2e-gce-gci
     context: pull-security-kubernetes-kubemark-e2e-gce-gci
-    rerun_command: "@k8s-bot pull-security-kubernetes-kubemark-e2e-gce-gci test this"
-    trigger: "@k8s-bot pull-security-kubernetes-kubemark-e2e-gce-gci test this"
+    rerun_command: "/test pull-security-kubernetes-kubemark-e2e-gce-gci"
+    trigger: "(^@k8s-bot pull-security-kubernetes-kubemark-e2e-gce-gci test this\r?$|^/test pull-security-kubernetes-kubemark-e2e-gce-gci\r?$)"
 
   - name: pull-security-kubernetes-node-e2e
     always_run: true
     context: pull-security-kubernetes-node-e2e
-    rerun_command: "@k8s-bot pull-security-kubernetes-node-e2e test this"
-    trigger: "@k8s-bot (pull-security-kubernetes-node-e2e |node e2e )?test this"
+    rerun_command: "/test pull-security-kubernetes-node-e2e"
+    trigger: "(^@k8s-bot (pull-security-kubernetes-node-e2e |node e2e )?test this\r?$|^/test( pull-security-kubernetes-node-e2e| node e2e)?\r?$)"
```

Comment:
Noted! If `always_run` equals to false, it will not support `/test` and `k8s-bot test this`.